[2025-03-28T08:13:33.980+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T08:13:34.107+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: daily_weather_etl.run_minio_script scheduled__2025-03-27T08:00:00+00:00 [queued]>
[2025-03-28T08:13:34.118+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: daily_weather_etl.run_minio_script scheduled__2025-03-27T08:00:00+00:00 [queued]>
[2025-03-28T08:13:34.122+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-28T08:13:34.136+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_minio_script> on 2025-03-27 08:00:00+00:00
[2025-03-28T08:13:34.142+0000] {standard_task_runner.py:72} INFO - Started process 562 to run task
[2025-03-28T08:13:34.146+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'daily_weather_etl', 'run_minio_script', 'scheduled__2025-03-27T08:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpuy454wro']
[2025-03-28T08:13:34.148+0000] {standard_task_runner.py:105} INFO - Job 138: Subtask run_minio_script
[2025-03-28T08:13:34.191+0000] {task_command.py:467} INFO - Running <TaskInstance: daily_weather_etl.run_minio_script scheduled__2025-03-27T08:00:00+00:00 [running]> on host 3c3c1f66214b
[2025-03-28T08:13:34.259+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='daily_weather_etl' AIRFLOW_CTX_TASK_ID='run_minio_script' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T08:00:00+00:00'
[2025-03-28T08:13:34.261+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-28T08:13:34.262+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-28T08:13:34.263+0000] {logging_mixin.py:190} INFO - Current task name:run_minio_script state:running start_date:2025-03-28 08:13:34.108974+00:00
[2025-03-28T08:13:34.264+0000] {logging_mixin.py:190} INFO - Dag name:daily_weather_etl and current dag run status:running
[2025-03-28T08:13:34.265+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T08:13:34.266+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-28T08:13:34.267+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/minio.py']
[2025-03-28T08:13:34.279+0000] {subprocess.py:99} INFO - Output:
[2025-03-28T08:13:36.999+0000] {subprocess.py:106} INFO - Setting default log level to "WARN".
[2025-03-28T08:13:37.000+0000] {subprocess.py:106} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-03-28T08:13:37.351+0000] {subprocess.py:106} INFO - 25/03/28 08:13:37 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-03-28T08:13:39.138+0000] {subprocess.py:106} INFO - Error: HTTPConnectionPool(host='api.weatherapi.com', port=80): Max retries exceeded with url: /v1/history.json?key=0184ac7ffd35494ca93123514251703&q=elche&dt=2025-03-27 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fc4192ef7f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
[2025-03-28T08:13:39.615+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-28T08:13:39.636+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T08:13:39.638+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=daily_weather_etl, task_id=run_minio_script, run_id=scheduled__2025-03-27T08:00:00+00:00, execution_date=20250327T080000, start_date=20250328T081334, end_date=20250328T081339
[2025-03-28T08:13:39.656+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-28T08:13:39.657+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-28T08:13:39.658+0000] {logging_mixin.py:190} INFO - Dag name:daily_weather_etl queued_at:2025-03-28 08:13:06.901413+00:00
[2025-03-28T08:13:39.659+0000] {logging_mixin.py:190} INFO - Task hostname:3c3c1f66214b operator:BashOperator
[2025-03-28T08:13:39.699+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T08:13:39.719+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-28T08:13:39.722+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
